### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by EricSmekens**
Updates of:
- WA-API-VERSION, patch to 2.29.3
- mysql, patch to 5.7.31

### CODE REVIEW

**Comment by EricSmekens** [comment]
*Signed CLA* - update

File: installation/docker-compose.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ volumes:
 
 services:
   db:
-    image: mysql:5.7.22
+    image: mysql:5.7.31
```
Comment (alanhhwong) [inline installation/docker-compose.yml:16]: @EricSmekens is there a specific reason for 5.7.31?  We have no tested this version thoroughly, but would love to hear if there's a specific need on your end for this version, or you merely picked the latest available

Comment (EricSmekens) [reply installation/docker-compose.yml:16]: I believe we have been using 5.7.27 for a while, but we will soon (coming weeks) start thorough testing with 5.7.31.
I will send a update here if we've completed the testing part.

Comment (alanhhwong) [reply installation/docker-compose.yml:16]: Got it.  We've tested 5.7.26 and are comfortable updating the scripts to support this version.  Will gladly accept this PR if you can help update the script to .26 for now while we work to test .31!

Comment (EricSmekens) [reply installation/docker-compose.yml:16]: Wil do that, makes perfect sense.

Next time, I'll update with a higher version, if we tested with that thoroughly.

Comment (EricSmekens) [reply installation/docker-compose.yml]: *Updated

**Pull request closed without merging**